localstack/config.py

```diff
@@ -625,6 +625,7 @@ def legacy_fallback(envar_name: str, default: T) -> T:
 
 
 # How to access LocalStack
+GATEWAY_LISTEN: List[HostAndPort]
```

Comment:
This should be covered by the types in the signature to `populate_legacy_edge_configuration`, no?